core/esmf-aspect-meta-model-java/src/test/java/org/eclipse/esmf/aspectmodel/edit/AspectChangeManagerTest.java

```diff
@@ -166,7 +166,7 @@ void testCreateFileWithElementDefinition() {
       final AspectModelFile aspectModelFile = RawAspectModelFileBuilder.builder()
             .sourceLocation( Optional.of( URI.create( "file:///temp/test.ttl" ) ) )
             .sourceModel( createModel( """
-                  @prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+                  @prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.2.0#> .
```

Comment:
Can this be changed to use `KnownVersion.getLatest()` so that it does not have to be changed again?